lib/verifiablePresentation/SSI.types.ts

```diff
@@ -1,8 +1,5 @@
 import { PresentationSubmission } from '@sphereon/pe-models';
-
-export interface CredentialSubject {
```

Comment:
I don;t think it is wise to remove this. 
The object has some well-known properties like the optional id. So instead of replacing it with unknown, we should extend this interface to accommodate for that.

Reply:
Fixed!